x-pack/plugin/src/test/resources/rest-api-spec/test/ml/delete_expired_data.yml

```diff
@@ -69,99 +69,3 @@ setup:
       catch: missing
       ml.delete_expired_data:
         job_id: not-a-job
----
-"Test delete expired data with job id":
```

Comment:
Delete expired data behavior was already tested (and more thoroughly)  in `org.elasticsearch.xpack.ml.integration.DeleteExpiredDataIT`